### PR TITLE
benchmarks: add script to run llama2 inference benchmarks

### PR DESCRIPTION
**Pull request by cota**

### CODE REVIEW

File: benchmarks/llama.py [hunk 1]
```diff
@@ -0,0 +1,236 @@
+import argparse
+import os
+import subprocess
+import json
+import datetime
+import re
+import sys
+
+
+def parse_args():
```
Comment (frgossen) [inline benchmarks/llama.py:10]: nit: Have this above the main for consistency with the other scripts?

File: benchmarks/llama.py [hunk 2]
```diff
@@ -0,0 +1,236 @@
...
+      xla = None if dynamo == 'inductor' else 'PJRT'
+      summary = f"{model_name} eval {backend} batch {args.batch_size}"
+
+      results_file = os.path.join(results_dir, 'results.jsonl')
+      if benchmark_has_already_run(results_file, model_name, xla, dynamo,
+                                   args.batch_size):
+        print(f"SKIP already completed benchmark -- {summary}")
+        continue
+
+      print(f"RUN {summary}")
```
Comment (frgossen) [inline benchmarks/llama.py:155]: Should we use a logger?

**Review by frgossen — COMMENTED** [review]
nice! ty!

File: benchmarks/llama.py (hunk 1 above)
Comment (cota) [reply benchmarks/llama.py]: Done.

File: benchmarks/llama.py (hunk 2 above)
Comment (cota) [reply benchmarks/llama.py]: I've added a bunch of debug logging printouts and also added the --log-level flag, just like in experiment_runner.py.

File: benchmarks/llama.py [hunk 3]
```diff
@@ -0,0 +1,236 @@
...
+  subprocess.check_call([
+      'git', 'clone', 'https://github.com/pytorch-tpu/llama.git', '--branch',
+      'llama2-google-next-inference', llama_dir
+  ])
+  subprocess.check_call(
+      ['pip', 'install', '-r',
+       os.path.join(llama_dir, 'requirements.txt')])
+  subprocess.check_call(['pip', 'install', '-e', llama_dir])
+
+  # TODO: remove once https://github.com/pytorch-tpu/llama/pull/47 is merged.
```
[the remark this answers is not in the document]
Comment (cota) [reply benchmarks/llama.py]: That PR was merged; I've dropped this hunk.

**Pull request merged**